drivers/adc/Kconfig.ad4130

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2025 Analog Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config ADC_AD4130
+	bool "ADI AD4130 Analog-to-Digital Converter"
+	default y
+	depends on DT_HAS_ADI_AD4130_ADC_ENABLED
+	select SPI
+	select ADC_CONFIGURABLE_INPUTS
+	help
+	  Enable the ad4130 adc
+
+config ADI_AD4130_ADC_ACQUISITION_THREAD_INIT_PRIO
+	int "ADC data acquisition thread priority"
+	default 0
```

Comment:
default of 0 is too high. Its the highest preemptive priority, please adjust it to be lower, especially given the thread spins constantly reading the I2C bus while waiting for sample to be ready. Set to 90 or default to SENSOR_INIT_PRIORITY (which is 90 by default)